core/storage/statistics/gae_models_test.py

```diff
@@ -186,6 +311,41 @@ def test_get_deletion_policy(self) -> None:
             .get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_create_event_models(self) -> None:
+        # No error raised while creating model.
+        stats_models.MaybeLeaveExplorationEventLogEntryModel.create(
+            'exp_id1', 1, 'state_name1', 'session_id1', 1.0, {},
+            feconf.PLAY_TYPE_NORMAL)
```

Comment:
Can we also check that the created model is correct? Ditto below

Reply:
Done!
I had to refactor the methods in gae_model.py to return `event entity id` so that model created could be validated since there were no existing methods to validate the model created. This also aligns with the other methods present in the file which return the `event entity id` to validate created model.